src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -352,4 +360,9 @@ export default compose(
             key: `${ONYXKEYS.COLLECTION.SELECTED_TAB}${CONST.TAB.RECEIPT_TAB_ID}`,
         },
     }),
+    withOnyx({
+        policy: {
+            key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY}${report.policyID}`,
+        },
+    }),
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/28182. The report object may be undefined and accessing the policyID on an undefined crashes the app.